WDAC-Policy-Wizard/app/Properties/Resources.resx

```diff
@@ -517,4 +517,23 @@ Are you sure you want to abandon the exception rule creation workflow?</value>
   <data name="UnsupportedCrypto_Error" xml:space="preserve">
     <value>A cryptographic algorithm that isn't supported by WDAC was found. Unfortunately, a publisher rule cannot be used to allow or deny this file. </value>
   </data>
-</root>
+  <data name="BINFileOpen_Error" xml:space="preserve">
+    <value>The WDAC Wizard could not open the WDAC XML file. The most likely reason is that the Wizard failed to build the policy file.
```

Comment:
English is difficult..

Reply:
I just had the strings switched. Fixed

Reply:
It's also the <value> that I find confusing. "The most likely reason is that the Wizard failed to build the policy file." What does that mean even? Maybe just tell them to check the log file for details?

Reply:
This error label can show up if the file is moved or some other file system issue. The likely culprit is that the file doesn't exist there since the Wizard failed to build it but its not the only reason